archive/storefront/blueprint/themes/global-variables.mdx

```diff
@@ -8,7 +8,7 @@ The list below is sorted by the number of times that the Blueprint base theme re
 
 | Variable | Description |
 |:---------|:------------|
-| `%%GLOBAL_ShopPath%%` | The full URL to the store without a trailing slash. If on an SSL based page, this will be the HTTPS version of the store path. |
+| `%%GLOBAL_ShopPath%%` | The full URL to the store without a trailing slash. If on an SSL-based page, this will be the HTTPS version of the store path. NOTE: This variable does not work for headless storefronts when linking to categories. In this case, you should use the URL instead of the store path variable. |
```

Comment:
Was this your change, @jpajar? Seems a little out of scope for our purposes here 😅

Reply:
Oh that's weird... it's not my changes but something must have happened with the rebase.